client/src/Movies/MovieList.js

```diff
@@ -34,7 +34,8 @@ export default class MovieList extends Component {
 function MovieDetails({ movie }) {
   const { title, director, metascore, stars } = movie;
   return (
-    <div className="movie-card">
+    <div  className="movie-card">
+    <Link to={`/Movies/id`}>Home
```

Comment:
```suggestion
    <Link to={`/Movies/${movie.id}`}>Home
```
the purpose here is to link each card to its own details of the movie, so you need to specify the id of that movie (see that you are passing the movie data on line 34? you will pull the id out of that data.